go.mod

```diff
@@ -1,138 +1,136 @@
 module github.com/rhobs/multicluster-observability-addon
 
-go 1.20
+go 1.21
 
 require (
-	github.com/imdario/mergo v0.3.12
-	github.com/openshift/cluster-logging-operator v0.0.0-20231130135759-9270994dc4bc
+	github.com/open-telemetry/opentelemetry-operator v0.92.0-2
+	github.com/openshift/cluster-logging-operator v0.0.0-20240119165623-e5237d4ec271
 	github.com/operator-framework/api v0.17.7
 	github.com/spf13/cobra v1.8.0
 	github.com/spf13/pflag v1.0.5
-	github.com/stretchr/testify v1.8.3
-	k8s.io/api v0.28.4
-	k8s.io/apimachinery v0.28.4
-	k8s.io/client-go v0.28.4
-	k8s.io/component-base v0.28.4
-	k8s.io/klog/v2 v2.100.1
-	k8s.io/utils v0.0.0-20230406110748-d93618cff8a2
+	github.com/stretchr/testify v1.8.4
+	go.opentelemetry.io/otel/exporters/otlp/otlptrace v1.19.0 // indirect
+	go.opentelemetry.io/otel/exporters/otlp/otlptrace/otlptracegrpc v1.19.0 // indirect
+	go.opentelemetry.io/otel/metric v1.22.0 // indirect
+	k8s.io/api v0.29.1
+	k8s.io/apimachinery v0.29.1
+	k8s.io/client-go v0.29.1
+	k8s.io/component-base v0.29.1
+	k8s.io/klog/v2 v2.120.1
+	k8s.io/utils v0.0.0-20231127182322-b307cd553661
 	open-cluster-management.io/addon-framework v0.8.0
 	open-cluster-management.io/api v0.12.0
-	sigs.k8s.io/controller-runtime v0.15.0
+	sigs.k8s.io/controller-runtime v0.17.0
```

Comment:
Watch out if you don't immediately need controller-runtime v0.17.0, keep the current definition or else prepare for any breaking changes: https://github.com/kubernetes-sigs/controller-runtime/releases/tag/v0.17.0

FYI you need to consider that the changes might impact the underlying addon framework that is internally a set of k8s controllers ;)

Reply:
I'm not sure why it is updated all the time by go mod tidy... I think because of the OTEL Operator